docs/Developers/Newcomers_Guide.rst

```diff
@@ -175,7 +175,7 @@ Now it is time to pick an issue.
 It is the best way to familiarise yourself with the codebase.
 You can view `all Newcomer issues on GitHub <https://coala.io/new>`_.
 
-.. note::
+.. caution::
```

Comment:
Maybe this is "important"?

We should definitely clear what messages are "caution" and what "important"